src/Infrastructure/Persistence/Migrations/20200115010414_Initial.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace Codidact.Infrastructure.Persistence.Migrations
+{
+    public partial class Initial : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "communities",
+                columns: table => new
+                {
+                    id = table.Column<long>(nullable: false)
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.SerialColumn),
+                    name = table.Column<string>(maxLength: 40, nullable: false),
+                    tagline = table.Column<string>(maxLength: 100, nullable: false),
+                    url = table.Column<string>(maxLength: 255, nullable: false),
+                    status = table.Column<int>(nullable: false, defaultValue: 1)
```

Comment:
Do we need `created_at`, `deleted_at` etc in here, or is that handled by inheritance elsewhere?

Reply:
We need it. AuditableEntity, which is supposed to hold all of the shared columns is currently empty. I figured it can be done in a later PR since the scope of this one is already way too big imo.

I am not sure about these initial migrations being here to be honest. I wanted to display that it works properly and the output it makes is correct.
But I am not sure if its better to create all of the initial entities first and then create an initial migration or to just create an entity each time and then add a migration.

Reply:
PRs should generally be complete features, even if that means they're big PRs. If it's related to adding EF into the project, it can go in here (except maybe creating all the entities - that can be done later).

Reply:
Fixed in b965a9decf80b7c98d8bfb715c7a429c7715581a